lib/Components/Divider.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+
+class DoubleDivider extends StatelessWidget{
+
+  @override 
+  Widget build(BuildContext context){
+    final instance = ScreenUtil.getInstance();
+    return Row(
```

Comment:
I'd wrap the row in a container setting the width of it to be 320 with instance set width

Reply:
Also on the row, I would recommend setting a CrossAxisAlignment of end on the row so that the dividers align around half ways up.